frontend/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "terminal.integrated.windowsEnableConpty": false
+}
```

Comment:
@sep2001aryan this code should not be committed together, it's not related to this PR

Reply:
Should i remove the vscode settings ?

Reply:
removed it